content/docs/command-reference/commit.md

```diff
@@ -42,10 +42,10 @@ scenarios are further detailed below.
 
 - In cases where we have previously executed a stage (either by writing
   `dvc.yaml` manually and using `dvc repro`, or with `dvc run`), but later
-  notice that some of the output files or directories it creates, which are
-  already in the <abbr>workspace</abbr>, are missing from `dvc.yaml` (`outs`
-  field). We can
-  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  notice that some of the existing dependencies of the stage or output
+  files/directories it creates, which are already in the <abbr>workspace</abbr>,
```

Comment:
Need to talk about files/dirs first, deps/outs later. Something like "...notice that some of the existing files/dirs used by the stage as dependencies, or created as outputs..."

Reply:
Done